app/lib/vendor_api/changes/gcse_reference_data.rb

```diff
@@ -0,0 +1,12 @@
+module VendorAPI
+  module Changes
+    class GcseReferenceData < VersionChange
```

Comment:
```suggestion
    class GcseSubjects < VersionChange
```
Although I wonder if we should add reference everywhere instead 🤔